application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -169,34 +169,45 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
 
 
         String applicationId;
+        String applicationName;
 
         if (applicationOption != null) {
-            applicationId = VC_APPLICATION_TO_ID.get(applicationOption.getAsString());
+            applicationName = applicationOption.getAsString();
+            applicationId = VC_APPLICATION_TO_ID.get(applicationName);
         } else {
             applicationId = idOption.getAsString();
+            applicationName = applicationId;
+
+            // Get the application name from the ID
+            for (var entry : VC_APPLICATION_TO_ID.entrySet()) {
```

Comment:
NIT: I am okay with `var` but we havent used it in the project so far. I guess most people would discourage using it in this specific case (since the types of the entry are not obvious)

Reply:
Would replacing `var` with `Map.Entry<String, String>` be better here then? 
Or is there a better alternative?

Reply:
I don't think there is any other option, without using the `forEach` method

Reply:
Changed in 107d309b978ad9aeb2783e0fb415b1ca750841e1